src/components/CohortBuilder/ParticipantsTableView/Test/SaveSetModal.test.tsx

```diff
@@ -6,18 +6,24 @@ import { jestPatchMatchMedia } from '../../../../utils';
 import { Provider } from 'react-redux';
 import configureStore from 'redux-mock-store';
 import thunk from 'redux-thunk';
-import { SaveSetState } from 'store/saveSetTypes';
+import { SaveSetState, UserSaveSets } from 'store/saveSetTypes';
 
 configure({ adapter: new Adapter() });
 
 const middleware = [thunk];
 const mockStore = configureStore(middleware);
+const userSaveSets = [] as UserSaveSets[];
```

Comment:
why casting here? can't you write this instead : userSaveSets: UserSaveSets = [] ?

Reply:
nope type is required.